src/celeritas/ext/g4vg/SolidConverter.cc

```diff
@@ -689,36 +689,40 @@ auto SolidConverter::convert_bool_impl(G4BooleanSolid const& bs)
     Array<G4VSolid const*, 2> solids{
         {bs.GetConstituentSolid(0), bs.GetConstituentSolid(1)}};
 
-    Transformation3D inverse;
-    {
-        // The "right" shape should be a G4DisplacedSolid which holds the
-        // matrix: replace solid and get inverse transform
-        CELER_EXPECT(!dynamic_cast<G4DisplacedSolid const*>(solids[0]));
-        CELER_EXPECT(dynamic_cast<G4DisplacedSolid const*>(solids[1]));
-        auto* displaced = static_cast<G4DisplacedSolid const*>(solids[1]);
-        solids[1] = displaced->GetConstituentMovedSolid();
-        inverse = convert_transform_(displaced->GetTransform().Invert());
-    }
-
     static Array<char const*, 2> const lr = {{"left", "right"}};
-    Array<Transformation3D const*, 2> transforms
-        = {{&Transformation3D::kIdentity, &inverse}};
     PlacedBoolVolumes result;
 
     for (auto i : range(solids.size()))
     {
-        // Convert solid
-        VUnplacedVolume const* converted = (*this)(*solids[i]);
+        G4VSolid const* solid = bs.GetConstituentSolid(i);
```

Comment:
```suggestion
        G4VSolid const* solid = solids[i];
```

Reply:
Whoops, meant to take that array out entirely, thanks! The only reason we preconstructed that is because the "displacement" seemed to be an asymmetric case which apparently it isn't.